src/WorkflowManager/Database/Repositories/TasksRepository.cs

```diff
@@ -76,34 +76,34 @@ private static async Task EnsureIndex(IMongoCollection<WorkflowInstance> workflo
             }
         }
 
-        public async Task<IList<TaskExecution>> GetAllAsync(int? skip, int? limit)
+        public async Task<(IList<TaskExecution>, long)> GetAllAsync(int? skip, int? limit)
         {
             try
             {
-                var builder = Builders<WorkflowInstance>.Filter;
-
-                var filter = builder.Eq("Tasks.Status", TaskExecutionStatus.Accepted);
-                filter &= builder.Ne("Tasks.Status", TaskExecutionStatus.Dispatched);
+                var filter = new FilterDefinitionBuilder<WorkflowInstance>().Where(x =>
+                x.Status != Status.Succeeded && x.Status != Status.Failed);
 
-                var result = await _workflowInstanceCollection.Aggregate()
-                    .Match(filter)
-                    .Unwind<WorkflowInstance, WorkflowInstanceTasksUnwindResult>(wf => wf.Tasks)
```

Comment:
if your no longer using WorkflowInstanceTasksUnwindResult might be worth removing it?

Reply:
Good shout, removed